libraries/chain/include/graphene/chain/son_wallet_object.hpp

```diff
@@ -21,6 +21,7 @@ namespace graphene { namespace chain {
          time_point_sec expires;
 
          flat_map<peerplays_sidechain::sidechain_type, string> addresses;
+         flat_map<peerplays_sidechain::sidechain_type, peerplays_sidechain::bytes> redeem_scripts;
```

Comment:
Redeem sctipt is already stored in address field, as address and a script are tightly paired. This field is not needed and breaks abstraction, by adding bitcoin specific field.

Reply:
ok